src/bin/packageJson.test.ts

```diff
@@ -0,0 +1,15 @@
+import { describe, expect, it } from "vitest";
+
+import { getVersionFromPackageJson } from "./packageJson.js";
+
+describe("Get version number from package.json", () => {
+	it("Returns a string representing a version number when given a valid package.json URL", async () => {
```

Comment:
[Naming] Nit: this repo normally goes with a slightly different convention...

* `describe`: just the name of the function (pending https://github.com/veritem/eslint-plugin-vitest/issues/247)
* `it`: lowercase _(otherwise this "X when Y" is phrasing great!)_

```suggestion
describe("getVersionFromPackageJson", () => {
	it("returns a string representing a version number when given a valid package.json URL", async () => {
```